core/include/drivers/serial.h

```diff
@@ -16,7 +16,9 @@ struct serial_chip {
 };
 
 struct serial_ops {
+	/* Must not be NULL */
```

Comment:
suggestion:
here:	`/* Mandatory handlers */`
and below: 	`/* Optional handlers */`